src/cmd/init.rs

```diff
@@ -74,6 +75,35 @@ pub enum InitOpt {
         #[structopt(name = "xpub-descriptor")]
         xpub: String,
     },
+    /// Initialize using a Coldcard SD card path.
+    ///
+    /// Requires a `coldcard-export.json` in this directory.
+    /// On Coldcard: Advanced -> MicroSD Card -> Export Wallet -> Generic JSON
+    /// Unsigned PSBTs will be saved to this SD card path, and read once signed.
+    Coldcard {
+        #[structopt(flatten)]
+        common_args: CommonArgs,
+        /// Coldcard SD card directory. PSBTs will be saved, signed, and loaded here.
+        #[structopt(parse(from_os_str))]
+        psbt_output_dir: PathBuf,
+        /// Instruct gun to use secret randomness from an exported deterministic entropy file.
+        /// On Coldcard: Advanced -> Derive Entropy -> 64-bytes hex.
+        /// Enter index 330 and press 1 to export to SD.
+        /// Gun will use entropy from drv-hex-idx330.txt for secret randomness.
+        #[structopt(long, short)]
+        entropy_export: bool,
```

Comment:
Seems more like we're instructing gun to import the entropy from the coldcard.
```suggestion
        import_entropy: bool,
```